tests/jobs/pyscal/test_pyscal.py

```diff
@@ -0,0 +1,205 @@
+"""Test module for ERT-Pyscal integration"""
+from __future__ import absolute_import
+
+import os
+
+import pytest
+
+import pandas as pd
+
+from semeio.jobs.scripts.fm_pyscal import run, rm_genkw_prefix
+
+TEST_PYSCAL_PATH, _ = os.path.split(os.path.abspath(__file__))
```

Comment:
Is this unused? If so you can just remove it.

Reply:
Removed